addon/-private/environment.js

```diff
@@ -0,0 +1,159 @@
+import Ember from 'ember';
```

Comment:
yeah I think that would help a good amount, taking this in all in one read is pretty tricky when it's all in a vertical line